gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -23,4 +23,92 @@ servers:
         default: http
         description: The schema of the Zeebe REST API server
 
-paths: {}
+paths:
+  /topology:
+    get:
+      summary: Get cluster topology.
+      description: Obtains the current topology of the cluster the gateway is part of.
+      responses:
+        '200':
+          $ref: '#/components/responses/TopologyResponse'
+
+components:
+  responses:
+    TopologyResponse:
+      description: Failed to accept request. No user task was found.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/TopologyResponse'
+
+  schemas:
+    TopologyResponse:
+      description: The response of a topology request.
+      type: object
+      properties:
+        brokers:
+          description: List of brokers that are part of this cluster
+          type: array
+          nullable: true
+          items:
+            $ref: "#/components/schemas/BrokerInfo"
+        clusterSize:
+          description: The number of nodes in the cluster
+          type: integer
+          nullable: true
+        partitionsCount:
+          description: how many partitions are spread across the cluster
+          type: integer
+          nullable: true
+        replicationFactor:
+          description: configured replication factor for this cluster
+          type: integer
+          nullable: true
+        gatewayVersion:
+          description: The version of the Zeebe Gateway
+          type: string
+          nullable: true
+    BrokerInfo:
+      description: Information on a broker node
+      type: object
+      properties:
+        nodeId:
+          description: unique (within a cluster) node ID for the broker
+          type: integer
+        host:
+          description: hostname of the broker
+          type: string
+        port:
+          description: port for the broker
+          type: integer
+        partitions:
+          description: list of partitions managed or replicated on this broker
+          type: array
+          items:
+            $ref: "#/components/schemas/Partition"
+        version:
+          description: broker version
+          type: string
+    Partition:
+      description: Information on a partition within a broker node
+      type: object
+      properties:
+        partitionId:
+          description: The unique ID of this partition
+          type: integer
+        role:
+          description:  Describes the Raft role of the broker for a given partition
+          type: string
+          enum:
+            - leader
+            - follower
+            - inactive
+        health:
+          description: Describes the current health of the partition
```

Comment:
:smile: What is the health property of the partition? It describes the health of the partition.

Well gosh, thanks! :smile: 

Let's write something a little more helpful for users :) If you want I can do that

Reply:
@npepinpe it would be better if you improve the docs there.